api/src/team/infrastructure/repositories/certification-center-invitation-repository.js

```diff
@@ -145,6 +146,20 @@ const markAsCancelled = async function ({ id }) {
   return _toDomain(certificationCenterInvitation);
 };
 
+/**
+ * @param {Object} params
+ * @param {Function} params.certificationCenterId
+ * @param {string} params.updatedAt
+ */
+
+const markAsCancelledByCertificationCenter = async function ({ certificationCenterId, updatedAt }) {
```

Comment:
Supprimer cette ligne vide : 
```suggestion
/**
 * @param {Object} params
 * @param {Function} params.certificationCenterId
 * @param {string} params.updatedAt
 */
const markAsCancelledByCertificationCenter = async function ({ certificationCenterId, updatedAt }) {
```